### ISSUE TITLE
List of requested of information about staking

### ISSUE THREAD

**Issue by alexlyp** [decred/decrediton#478]
I'm trying to tabulate a list of information people would like about their staking set ups.  Once we feel like this information is complete (after reviewing each entry for correctness and whether it's actually possible to generate), then we can proceed to have the UX/UI designers create a rough design for a new staking page in PM and decrediton.

- History of Ticket Purchases
  - Ticket prices
  - Fees paid
  - Number of tickets
- History of Voting
  - Rewards received
  - Pool fees paid
- Overall profitability/return (month/month, year/year)
- Ticket price history (similar to what is seen on dcrstats)
- History of Missed/Expired/Revoked
- Balance history 
- Current Immature/Live tickets
  - Age of current tickets (to see how close they are to expiry)
- Past vote choices (and maybe results of those votes)
- Show possible return at current ticket price, current balance and current reward?

Feel free to leave any ideas/additions to this below.

**Comment by jcvernaleo** [comment]
Not sure how easy it is, but age of oldest current ticket might be worth knowing.

**Comment by alexlyp** [comment]
Ah good call, will add that.

**Comment by pvtwarren** [comment]
Perhaps something like a projected annual total return return rate? However this requires a lot of assumptions so not sure if feasible.

**Comment by alexlyp** [comment]
@pvtwarren yah i would like to do something like that.  definitely tough and there are a lot of assumptions, but i think we could use total balance, average ticket price, average fee and reward over that period to get a pretty accurate number.

**Comment by alexlyp** [comment]
Much of this information has been added with the recent addition of the MyTickets page.  I will be opening up a new issue to discuss which statistics we are going to include in the upcoming Statistics page.
